aot/src/main/java/com/dylibso/chicory/aot/AotMethods.java

```diff
@@ -26,6 +26,14 @@ public static long[] callIndirect(long[] args, int typeId, int funcId, Instance
         return instance.getMachine().call(funcId, args);
     }
 
+    public static long[] callHostFunction(Instance instance, int funcId, long[] args) {
+        var imprt = instance.imports().function(funcId);
+        if (imprt == null) {
```

Comment:
This check shouldn't be needed, as we validate the imports in `com.dylibso.chicory.runtime.Instance.Builder#mapHostImports`

So this could be inlined
```java
return instance.imports().function(funcId).handle().apply(instance, args);

Reply:
good catch!